opentelemetry-tracing/opentelemetry-dotnet-manual/src/ClientExample/ClientExample.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
NET Core 3.1 will be EOL December 2022. https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core
I think that we should crate new samples based on .NET 6.

(No only this file, full search 3.1).

Reply:
Leaving out cloud foundry example - let's look at it separately. For the others done via 838f401